src/Repository/ItemRepository/item.ts

```diff
@@ -13,4 +13,24 @@ export class ItemRepository {
         return error;
       });
   }
+ 
+  public async AllItem(): Promise<any> {
+    let result =  Item.find({});
+    
+  }
+  public async SingleItem(itemId: any): Promise<any> {
+    const item = await Item.findById(itemId)
+    return item
+  }
+ 
+  public async editItem(itemId: any): Promise<any> {
+    const item = await Item.findById(itemId)
+    return item
+  }
+
+  public async removeItem(Id: any): Promise<any> {
+    const item= await Item.findById(Id)
+    if (!item) return null;
```

Comment:
You don't need to check for nullity here..Just return the item. Check for nullity in service

Reply:
Done